### DISCUSSION TITLE
openssl updates

### DISCUSSION THREAD

**Discussion by rootnk** [quictls/openssl#139] [General]
When will openssl 3.0.11 + quictls
and
openssl 3.1.3 + quictls be available?

**Comment by jonahyongNTU** [comment]
Yes, please. When will OpenSSL 3.0.11+QuickTLS be released?
My company security scanning software (Tenable) is detecting version 3.0.10 as high severity vulnerability.

Thanks,
Jonah

**Comment by tmshort** [comment]
Apologies for the delay.
We have new releases based on this week's upstream OpenSSL releases.
All prior releases have also been tagged and released.
